src/Parsers/ParserCreateQuery.h

```diff
@@ -211,6 +211,18 @@ bool IParserColumnDeclaration<NameParser>::parseImpl(Pos & pos, ASTPtr & node, E
             if (!s_null.ignore(pos, expected))
                 return false;
             null_modifier.emplace(false);
+
+            pos_before_specifier = pos;
+            if (s_default.ignore(pos, expected) || s_materialized.ignore(pos, expected) ||
```

Comment:
Need to check does `default` has been parsed, current
 ```sql
CREATE TEMPORARY TABLE
(
    x INT DEFAULT 1 NOT NULL DEFAULT 2
)
``` 
seems also can be succussfully parsed.

Reply:
yeah. But it also can be parsed in Mysql XD.

Reply:
Hi, @evillique, what do you think?